solutions/quickstart/main.tf

```diff
@@ -0,0 +1,165 @@
+  crn     = var.existing_secrets_manager_instance_crn
+}
+
+# Create a secret group to place the certificate if provisioning a new certificate
+module "secrets_manager_secret_group" {
+  source                   = "terraform-ibm-modules/secrets-manager-secret-group/ibm"
+  version                  = "1.2.2"
+  region                   = module.existing_sm_crn_parser.region
+  secrets_manager_guid     = module.existing_sm_crn_parser.service_instance
+  secret_group_name        = var.prefix != null ? "${var.prefix}-cert-secret-group" : "cert-secret-group"
+  secret_group_description = "secret group used for private certificates"
+  providers = {
+    ibm = ibm.ibm-sm
+  }
+}
+
+# Create private certificate to use for VPN server
+module "secrets_manager_private_certificate" {
+  source                 = "terraform-ibm-modules/secrets-manager-private-cert/ibm"
+  version                = "1.3.1"
+  cert_name              = var.prefix != null ? "${var.prefix}-cts-vpn-private-cert" : "cts-vpn-private-cert"
+  cert_description       = "private certificate for client to site VPN connection"
+  cert_template          = var.certificate_template_name
+  cert_secrets_group_id  = local.secrets_manager_secret_group_id
+  cert_common_name       = var.cert_common_name
+  secrets_manager_guid   = module.existing_sm_crn_parser.service_instance
+  secrets_manager_region = module.existing_sm_crn_parser.region
+  providers = {
+    ibm = ibm.ibm-sm
+  }
+}
+
+##############################################################################
+# Deploy client-to-site in a dedicated subnet in the VPC
+##############################################################################
+locals {
+  vpc_region      = module.existing_vpc_crn_parser.region
+  existing_vpc_id = module.existing_vpc_crn_parser.resource
+  subnet_ids      = [ibm_is_subnet.client_to_site_subnet_zone_1.id]
+  zone_1          = "${local.vpc_region}-1" # hardcode to first zone in region
+  target_ids      = [module.vpn.vpn_server_id]
+  vpn_server_routes = {
+    "vpc-10" = {
+      destination = "10.0.0.0/8"
+      action      = "deliver"
+    },
+    # Add route for PaaS IBM Cloud backbone. This is mostly used to give access to the Kube master endpoints.
+    "vpc-166" = {
+      destination = "166.8.0.0/14"
+      action      = "deliver"
+    }
+  }
+}
+
+module "existing_vpc_crn_parser" {
+  source  = "terraform-ibm-modules/common-utilities/ibm//modules/crn-parser"
+  version = "1.0.0"
+  crn     = var.existing_vpc_crn
+}
+
+resource "ibm_is_vpc_address_prefix" "client_to_site_address_prefixes_zone_1" {
+  name = var.prefix != null ? "${var.prefix}-client-to-site-address-prefixes-1" : "client-to-site-address-prefixes-1"
+  zone = local.zone_1
+  vpc  = local.existing_vpc_id
+  cidr = "10.10.40.0/24"
+}
+
+resource "ibm_is_network_acl" "client_to_site_vpn_acl" {
+  name = var.prefix != null ? "${var.prefix}-client-to-site-acl" : "client-to-site-acl"
+  vpc  = local.existing_vpc_id
+  rules {
```

Comment:
same as in standard version, you might need to add deny-all rules to the end of these inbount/outbound sets.